e2e/bash/test_bash_shell_integration.py

```diff
@@ -69,3 +69,11 @@ def test_wordbreaks_chars(complgen_binary_path: Path):
     with temp_usage_file_path(complgen_binary_path, GRAMMAR, 'mygrep') as usage_file_path:
         input = r'''COMP_WORDS=(mygrep --color=); COMP_CWORD=1; _complgen_jit_mygrep; printf '%s\n' "${COMPREPLY[@]}"'''
         assert get_sorted_bash_completions(usage_file_path, input) == sorted(['--color=always', '--color=never', '--color=auto'])
+
+
+# This test also needs to be tested manually in an interactive shell
+def test_colons(complgen_binary_path: Path):
+    GRAMMAR = 'colontest (b:c | b:d);'
+    with temp_usage_file_path(complgen_binary_path, GRAMMAR, 'colontest') as usage_file_path:
+        input = r'''COMP_WORDS=(colontest b); COMP_CWORD=1; _complgen_jit_colontest; printf '%s\n' "${COMPREPLY[@]}"'''
+        assert get_sorted_bash_completions(usage_file_path, input) == sorted(['b:c', 'b:d'])
```

Comment:
Similarly to above: it would be great to port this into a pty e2e test similarly to how zsh is already tested: https://github.com/adaszko/complgen/blob/124bc1721d3547386cb1c8dda24e63dc6ba85153/e2e/zsh/test_zsh_pty.py